homeassistant/components/tractive/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Tractive",
```

Comment:
This is a brandname, I think that doesn't need to be translated in this case.

```suggestion
```

Please update the translation file after change the strings